Lib/test/test_importlib/extension/test_loader.py

```diff
@@ -145,6 +146,15 @@ def test_reload(self):
             importlib.reload(module)
             self.assertIs(ex_class, module.Example)
 
+    def test_reload_faliure(self):
+        '''Test that reload throws ModuleNotFounderror if reloading'''
```

Comment:
Actually the whole module is a mess when it comes to docstrings.

Actually use a `#` comment instead of a docstring, and it can take more than a single line. Also make sure to end it in a period.